src/main/java/com/team2813/Constants.java

```diff
@@ -52,16 +52,17 @@ private DriverConstants() {
 
     public static final CommandPS4Controller DRIVER_CONTROLLER = new CommandPS4Controller(0);
     public static final Trigger SYSID_RUN = DRIVER_CONTROLLER.cross().and(() -> false);
-    public static final Trigger SLOWMODE_BUTTON = DRIVER_CONTROLLER.R3();
-    public static final Trigger PLACE_CORAL = DRIVER_CONTROLLER.R2();
+    public static final Trigger SLOWMODE_BUTTON = DRIVER_CONTROLLER.L1();
+    public static final Trigger PLACE_CORAL = DRIVER_CONTROLLER.R1();
     public static final Trigger RESET_POSE = DRIVER_CONTROLLER.triangle();
 
-    public static Trigger AUTO_ALIGN_LEFT = DRIVER_CONTROLLER.L1();
-    public static Trigger AUTO_ALIGN_RIGHT = DRIVER_CONTROLLER.R1();
+    public static Trigger AUTO_ALIGN_LEFT = DRIVER_CONTROLLER.povUp();
+    public static Trigger AUTO_ALIGN_RIGHT = DRIVER_CONTROLLER.povDown();
```

Comment:
There is nothing bound to povLeft & povRight, so we might want to use those so the bindings are more intuitive
```suggestion
    public static Trigger AUTO_ALIGN_LEFT = DRIVER_CONTROLLER.povLeft();
    public static Trigger AUTO_ALIGN_RIGHT = DRIVER_CONTROLLER.povRight();
```
If the intent is to make the buttons not usable, then you can do the following to ensure they don't get used:
```suggestion
    public static Trigger AUTO_ALIGN_LEFT = DRIVER_CONTROLLER.povLeft().and(() -> false);
    public static Trigger AUTO_ALIGN_RIGHT = DRIVER_CONTROLLER.povRight().and(() -> false);
```